server/datastore/mysql/mysql.go

```diff
@@ -820,3 +820,25 @@ func hostSearchLike(sql string, params []interface{}, match string, columns ...s
 	}
 	return base, args
 }
+
+func (ds *Datastore) InnoDBStatus(ctx context.Context) (string, error) {
+	status := struct {
+		Type   string `db:"Type"`
+		Name   string `db:"Name"`
+		Status string `db:"Status"`
+	}{}
+	err := ds.writer.GetContext(ctx, &status, "show engine innodb status")
```

Comment:
`ds.reader` (we need a lint rule :)

Reply:
Or add a comment on why we would use `ds.writer` (same as in `Datastore.DBLocks`).

Reply:
In this case, I think we need to go for the writer, as it's the most important node. I don't think these queries return data of a cluster, but for the node they run in.